libs/server/Storage/Functions/MainStore/RMWMethods.cs

```diff
@@ -413,6 +415,28 @@ private bool InPlaceUpdaterWorker(ref SpanByte key, ref SpanByte input, ref Span
                     rmwInfo.Action = RMWAction.ExpireAndStop;
                     return false;
 
+                case RespCommand.GETEX:
+                    CopyRespTo(ref value, ref output);
+
+                    if (input.ExtraMetadata > 0)
```

Comment:
Shouldn't you check for !isPersist here?

Reply:
I don't have to because if there is ExtraMetadata then that means the user has provided the expiration time so it can't be a Persist options, so there is no point in checking for `!isPersist`